idowuadeleke/BANKA
# Feat(SignIn):connect to Database

### What does this PR do?
Persist user sign in data to a database

### Description of Task to be completed?
- Write unit test cases for user sign in connected to database

- create API endpoint that takes user signin input and stores them in a database

- replace sign up with data structure with sign in with database

### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ft-165434489-User-Signin-Database

- run `npm install`

- run `npm run start`

- make a request to `POST localhost:3000/api/v1/auth/signin`
  NB see screenshot below on the request data to pass

### Any background context you want to provide?
NA
### What are the relevant pivotal tracker stories?
[#165434489](https://www.pivotaltracker.com/story/show/165434489)

### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40268693/56326583-7e040400-616e-11e9-9096-62aa9c81736c.png)
